CHANGELOG.md

```diff
@@ -1,5 +1,21 @@
 # Changelog
 
+## 0.58.0      2024-08-27
+
+* Log Insights:
+  - New log parsing mechanism that is more performant and allows
+    for arbitrary log_line_prefix settings. The new parsing mechanism
+    is the default, but you can set `db_log_line_prefix = legacy` in
+    the config file or `DB_LOG_LINE_PREFIX=legacy` in the environment
+    to revert to the old mechanism.
+* RDS:
+  - Update AWS SDK to v1.55.3
```

Comment:
This affects https://github.com/pganalyze/pganalyze-docs/pull/250#discussion_r1578972209 , but is that something we want to mention here before we finish those docs?

Reply:
I don't think we need to advertise that we started supporting it. It's still nice to update docs (still in my TODO list but a little bit in the bottom), but yeah I'd say no need to mention.

Reply:
Maybe we can just note that EKS pod identity is now supported? I think for those that care about it they'd appreciate knowing its supported, and would if needed reach out if unclear how to utilize it.

Reply:
I wasn't sure if the setup flow goes well as I wasn't able to complete the test back then, hence the hesitation.
That said, it'll likely go well and the fact that the updated SDK version does support EKS Pod Identity credentials so mentioning it will not going to be a lie, so maybe it's fine to mention it.
FYI, I plan to test and update the docs after the collector release, as testing before the release is going to be a PITA.

Reply:
I've changed this to
```
* RDS: Update AWS SDK to v1.55.3
  - This allows using EKS pod identity; documentation coming soon
```